pkgs/development/libraries/p11-kit/default.nix

```
@@ -16,17 +16,12 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev"];
   outputBin = "dev";
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig which ];
+  nativeBuildInputs = [ meson pkgconfig which ninja ];
   buildInputs = [ gettext libffi libiconv libtasn1 ];
 
-  autoreconfPhase = ''
-    NOCONFIGURE=1 ./autogen.sh
-  '';
-
-  configureFlags = [
-    "--sysconfdir=/etc"
-    "--localstatedir=/var"
-    "--without-trust-paths"
+  mesonFlags = [
+    "-Dsystem_config=${placeholder "out"}/etc"
```

This should probably be 
```suggestion
    "-Dsystem_config=${placeholder "out"}/etc/pkcs11"
```

https://github.com/p11-glue/p11-kit/blob/ef2716d18e1098effb8f3a90653c6264b2fec426/meson.build#L288-L291

---

No matter what, we will need to patch https://github.com/p11-glue/p11-kit/blob/17b2edb1354c5d5080172af45ce55408c5e51d59/p11-kit/meson.build#L27-L28 to `/etc/…`, or (probably better) drop `-Dsystem_config` and patch https://github.com/p11-glue/p11-kit/blob/17b2edb1354c5d5080172af45ce55408c5e51d59/p11-kit/meson.build#L320 to `$out/etc`.